feathr_project/feathr/_feature_registry.py

```diff
@@ -915,18 +920,80 @@ def search_input_anchor_features(self,derived_guids,feature_entity_guid_mapping)
         return result
 
 
-
+    def _correct_function_identation(self, user_func: str) -> str:
```

Comment:
put this in the preprocessing_py_udf module?

Reply:
I thought about it as well but think it's cleaner to put it in the registry module, i.e. the functions that are retrieved from registry should have the correct indentation.